README.md

```diff
@@ -2,24 +2,250 @@
+
+The ASC meets quarterly, with at least one quarterly meeting each year to be held "in-person" at an agreed-upon location unless the ASC Members vote otherwise. A vote to "virtualize" the in-person meeting can only apply to that given year. Other quarterly meetings may be held by teleconference in accordance with the desires of ASC members. ASC quarterly meetings are conducted according to the following schedule:
+
+* 3rd week of January
+* 3rd week of April
+* 3rd week of July
+* 3rd week of October
+
+ASC Members may, if desired, vote to alter the timing of the meetings in a given year to accommodate Member schedules, subject to the voting rules described below. Changes to a given meeting's dates must be determined no later than the prior quarterly meeting.
+
+The Co-Chairs shall include links to each proposed modification to the Standard up for consideration during that meeting in the published agenda.
+
+#### Current ASC Members
+
+<div  align="center">
+<table>
+ <tr>
+  <td>Intel (Ralph Castain)*</td>
+  <td>IBM (Josh Hursey)**</td>
+  <td>Mellanox (Artem Polyakov)</td>
+ </tr>
+ <tr>
+  <td>UnivTenn-Knoxville (Aurelien Bouteiller)</td>
+  <td>LANL (Howard Pritchard)</td>
+  <td>ORNL (Thomas Naughton)</td>
+ </tr>
+ <tr>
+    <td>Stony Brook Univ (Anthony Curtis)</td>
+    <td>ARM (Dirk Schubert)</td>
+</tr>
+<tr>
+    <td>Altair (Mike Karo)</td>
+    <td>RogueWave (John DelSignore)</td>
+    <td>Sylabs (Geoffroy Vallee)</td>
+</tr>
+</table>
+</div>
+\*  Co-Chair (even year) &nbsp;&nbsp;&nbsp;&nbsp; \** Co-Chair (odd year)
+
+### General voting rules
+
+The philosophy and goal of the ASC is to reach consensus on all, if not most, issues. In the event that full consensus cannot be reached, the ASC will use the following rules:
+
+* Each active ASC Member organization will receive one vote. All votes are required to pass with 2/3 majority. A quorum (>50%) of Members must be present (either physically or virtually). Abstentions will not be counted in the total votes when calculating whether there is a 2/3 majority.
+
+* A Member may submit a vote _in absentia_ to the ASC Co-Chairs in advance of the meeting if attendance by a representative is not possible (e.g., a quarterly meeting that falls during a national holiday period), but each ASC Member organization can only cast one vote. This prevents a Member from voting on behalf of its own entity and acting as a proxy for another member. _In absentia_ votes are counted by the Co-Chairs as if the member were present (thus counting towards the meeting's quorum requirement), but are recorded as having been voted _in absentia_ in the official tally.
+
+Formal votes may, if requested by at least one Member, be conducted as "secret ballots" to abate concerns over customer and/or competitor perceptions. All votes are tallied by the Co-Chairs and recorded in the minutes of the meeting.
```

Comment:
The secret ballot seems strange to me in a public forum like this. I think this point needs discussion

Reply:
Sure - it is intended to help vendors when confronted by something a customer (either current or potential) really wants but the vendor opposes.